### PR TITLE
new Hempcretesand & Hempcrete recipe

### PR DESCRIPTION
**Pull request by Darkmaxsas**
suggestion for the new Chisel blocks : HempcreteSand and Hempcrete

### CODE REVIEW

**Comment by Darkmaxsas** [comment]
Hempcrete
HempcreteSand

File: src/main/java/com/dreammaster/gthandler/recipes/MixerRecipes.java [hunk 1]
```diff
@@ -1,21 +1,9 @@
...
-import static gregtech.api.enums.Mods.GTPlusPlus;
-import static gregtech.api.enums.Mods.Genetics;
-import static gregtech.api.enums.Mods.IndustrialCraft2;
-import static gregtech.api.enums.Mods.Natura;
-import static gregtech.api.enums.Mods.OpenComputers;
-import static gregtech.api.enums.Mods.PamsHarvestCraft;
-import static gregtech.api.enums.Mods.Thaumcraft;
-import static gregtech.api.enums.Mods.TinkerConstruct;
-import static gregtech.api.enums.Mods.Witchery;
+import static gregtech.api.enums.Mods.*;
```
Comment (chochem) [inline src/main/java/com/dreammaster/gthandler/recipes/MixerRecipes.java:4]: its our policy not to do this. maybe adjust you IDE if it was automatic.

File: src/main/java/com/dreammaster/gthandler/recipes/MixerRecipes.java [hunk 2]
```diff
@@ -1503,5 +1491,12 @@ public void run() {
                     20,
                     16);
         }
+
+        if (Chisel.isModLoaded()) {
+            GT_Values.RA.stdBuilder().itemInputs(GT_ModHandler.getModItem(Chisel.ID, "hempcretesand", 1L))
```
Comment (chochem) [inline src/main/java/com/dreammaster/gthandler/recipes/MixerRecipes.java:1511]: would I be correct in assuming there is a chisel PR following or do you want both recipes?

Comment (Darkmaxsas) [reply src/main/java/com/dreammaster/gthandler/recipes/MixerRecipes.java:1511]: there will be one following in a min

Comment (Darkmaxsas) [reply src/main/java/com/dreammaster/gthandler/recipes/MixerRecipes.java:1511]: https://github.com/GTNewHorizons/Chisel/pull/31

**Comment by Darkmaxsas** [comment]
hempcretesand craft
forgot to mention the new crafting recipe

**Review by chochem — APPROVED** [review]
sure I guess, that should make everyone happy then hopefully.

File: src/main/java/com/dreammaster/gthandler/recipes/MixerRecipes.java (hunk 1 above)
Comment (Darkmaxsas) [reply src/main/java/com/dreammaster/gthandler/recipes/MixerRecipes.java]: mb, i didn't know. I will disable it in my idea for next time.

**Review by boubou19 — APPROVED** [review]

**Pull request merged**